immersion/kubeflow_pipelines/walkthrough/solutions/lab-01_vertex.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>Done!</p>

Reply:
<p>My bad, I should have wrote <code>1e-4</code>  etc, but I believe we do not need <code>1.0e-4</code>,  we can just write <code>1e-4</code> (both in text and code), as the exponent is negative, it will be <code>float</code>  class anyway.</p>

Reply:
<p>oh i am wrong, YAML needs this, maybe then 0.0001 is fine too or this is lgtm</p>